.github/workflows/ovc.yml

```diff
@@ -0,0 +1,53 @@
+name: OVC
+on:
+  push:
+    paths:
+      - 'tools/ovc/**'
+      - '.github/workflows/ovc.yml'
+    branches:
+      - 'master'
+      - 'releases/**'
+  pull_request:
+    paths:
+      - 'tools/ovc/**'
+      - '.github/workflows/ovc.yml'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  Pylint-UT:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Clone OpenVINO
+        uses: actions/checkout@v4
+
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
+
+      - name: Cache pip
+        uses: actions/cache@v4
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('src/bindings/python/requirements*.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+            ${{ runner.os }}-
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip setuptools
+          # For UT
+          pip install unittest-xml-reporting==3.0.2
+          pip install pylint>=2.7.0
+          pip install pyenchant>=3.0.0
+
+          pip install -r requirements.txt
+        working-directory: src/bindings/python/
+
```

Comment:
Pylint check is optinal now to merge in the master.
Can we make it required check for merge?

Reply:
Discussed it with @akladiev, yes we can add this workflow to required checks few days after merge of this PR.